toolkits/global/packages/global-status-message/README.md

```diff
@@ -119,8 +101,22 @@ To trigger announcement of a new status message, the live region must be added t
 </div>
 ```
 
-You can achieve the initial state (without the contents) by omitting the `iconURL` and `message` properties in the data. 
+#### Status changed
+
+If the status message changes, update the `message` and `type` properties. This means the screen reader will announce the new message. 
 
 ### Focus
 
-Where an `id` is supplied in the data, the status message can be focused programmatically. Focusing the message ensures it is visible in the viewport _and_ announces it in screen reader software. The status message is likely to also be the optimal focus position for (continued) keyboard interaction.
+Supply an `id` in the data to make the status message focusable programmatically.
+
+Focusing the message ensures that it is visible in the viewport and a screen reader will announce it.
```

Comment:
I don't think it's clear, on reflection, when to focus and when to just rely on the live region. 

```suggestion
Focusing the message ensures it is visible in the viewport and a screen reader will announce it. If it is not appropriate to move focus to the message, rely on the live region for announcement in screen readers.
```